extensions.html

```diff
@@ -1,920 +1,1152 @@
+          leftmost coordinate and 1.0 references the rightmost coordinate.
+          Along the y-axis, -1.0 references the topmost coordinate and 1.0
+          references the bottommost coordinate. MUST be a two-element array.
+        </dd>
+        <dt>
+          <dfn>surfaceDimensions</dfn>
+        </dt>
+        <dd>
+          Width and height of the touch surface in integer units. When not
+          <code>null</code>, MUST be a two-element array.
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>
+        Partial <dfn>Gamepad</dfn> Interface
+      </h2>
+      <p>
+        This partial interface supplements the Gamepad interface described in
+        the main <a href="https://w3c.github.io/gamepad/">Gamepad spec</a>.
+      </p>
+      <pre class="idl">
+        partial interface Gamepad {
+          readonly attribute GamepadHand hand;
+          readonly attribute FrozenArray&lt;GamepadHapticActuator&gt; hapticActuators;
+          readonly attribute GamepadPose? pose;
+          readonly attribute FrozenArray&lt;GamepadTouch&gt;? touchEvents;
+          [SameObject] readonly attribute GamepadHapticActuator vibrationActuator;
+        };
+      </pre>
+      <p>
+        Instances of {{Gamepad}} are created with the internal slots described
+        in the following table:
+      </p>
+      <table class="simple">
+        <tr>
+          <th>
+            Internal slot
+          </th>
+          <th>
+            Initial value
+          </th>
+          <th>
+            Description (non-normative)
+          </th>
+        </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for="Gamepad">[[\hand]]</dfn>
+          </td>
+          <td>
+            `undefined`
+          </td>
+          <td>
+            Indicates the hand the controller is held in.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for="Gamepad">[[\hapticActuators]]</dfn>
+          </td>
+          <td>
+            `undefined`
+          </td>
+          <td>
+            List of all the haptic actuators in the gamepad.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for="Gamepad">[[\pose]]</dfn>
+          </td>
+          <td>
+            `null`
+          </td>
+          <td>
+            The current pose of the gamepad.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <dfn data-dfn-for="Gamepad">[[\touchEvents]]</dfn>
+          </td>
+          <td>
+            <code>null</code>
+          </td>
+          <td>
+            List of generated touch events.
+          </td>
+        </tr>
+      </table>
+      <dl data-dfn-for="Gamepad">
+        <dt>
+          <dfn>hand</dfn>
+        </dt>
+        <dd>
+          An enumeration that indicates which hand the controller is being held
+          in or is most likely to be held in. `"left"`, `"right"`, or `""`.

```

Comment:
Would "both" be better than ""?

Reply:
The enumeration is better described above in the document. I changed this to refer to the above definition. I think "" can mean "both", "either", or "NA", and perhaps it only matters to the developer if it is "left" or "right".
Perhaps @nondebug  has more insight.